app/controllers/cards_controller.rb

```diff
@@ -1,9 +1,10 @@
 class CardsController < ApplicationController
-  before_action :require_login, :only => :new
+  # before_action :require_login, :only => :new
+  before_action :authenticate, :only => :new
```

Comment:
we had two ways. We can leave old monolith controllers and our rails monolith app will work, and api/v1 will work. Or we can destroy all old routes, views and other related to monolith app, and make these app only for api. WYT?